src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,4 +1,10 @@
+import { FC } from 'react';
+import { User } from '../../types/User';
+import './UserInfo.scss';
+
 // Add the required types and props
-export const UserInfo = () => (
-  <>UserInfo markup</>
+export const UserInfo: FC<{ user:User }> = ({ user }) => (
```

Comment:
Today your component receives just the user but tomorrow there can be more props
```suggestion
export const UserInfo: FC<interface> = ({ user }) => (
```